calico.py

```diff
@@ -162,13 +175,29 @@ def add(self):
 
         _log.debug("Checking for existing Calico endpoint")
         endpoint = self._get_endpoint()
-        if endpoint:
+        if endpoint and not self.k8s_namespace:
+            # Not running under Kubernetes.
             # This endpoint already exists, add it to another network.
             _log.info("Endpoint for container exists - add to new network")
             output = self._add_existing_endpoint(endpoint)
+        elif endpoint and self.k8s_namespace:
+            # Running under Kubernetes and we've received a create for 
```

Comment:
It is sort of both.

With the switch to using pod name and namespace for the workload ID, we need to handle this case.  If a host gets restarted, for example, we'll try to set up networking for a workload that already exists.

This make sure we don't have duplicate endpoints for the same workload.